Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM fedora:37
+FROM  registry.access.redhat.com/ubi9:latest
 
+RUN dnf install -y --nodocs skopeo
```

Comment:
We can save some space if we add a dnf clean all after this install.
test1 is how you have it test2 is with dnf clean all after skopeo install
```
$ sudo podman image ls
REPOSITORY                                   TAG              IMAGE ID      CREATED         SIZE
localhost/test2                              latest           61f35dc95cdc  31 seconds ago  1.18 GB
localhost/test1                              latest           9b17aca28a12  4 minutes ago   1.19 GB
```

Reply:
Updated, @dry923 ptal